### PR TITLE
feat: archive is now part of project feature list

### PR DESCRIPTION
**Pull request by sjaanus**
image

### CODE REVIEW

File: frontend/src/component/project/Project/PaginatedProjectFeatureToggles/ProjectOverviewFilters.tsx
Comment (thomasheartman) [inline frontend/src/component/project/Project/PaginatedProjectFeatureToggles/ProjectOverviewFilters.tsx:1]: Not something that's blocking for now, but is "Show only archived is true" really the best way to phrase it / display it to users? 🤔

File: frontend/src/component/project/Project/PaginatedProjectFeatureToggles/hooks/useRowActions.tsx
Comment (thomasheartman) [inline frontend/src/component/project/Project/PaginatedProjectFeatureToggles/hooks/useRowActions.tsx:1]: Where do we show these? Is this the bar in the screenshot or is it a row? Just wondering whether we're always showing the option to revive and delete or if it's done conditionally somehow. I'm assuming that it's only shown where it makes sense, but I'm not seeing exactly where that's happening 😅

**Review by thomasheartman — APPROVED** [review]

File: frontend/src/component/project/Project/PaginatedProjectFeatureToggles/ProjectOverviewFilters.tsx
Comment (sjaanus) [reply frontend/src/component/project/Project/PaginatedProjectFeatureToggles/ProjectOverviewFilters.tsx:1]: Most easiest  to understand, we can change it in overhaul.

File: frontend/src/component/project/Project/PaginatedProjectFeatureToggles/hooks/useRowActions.tsx
Comment (sjaanus) [reply frontend/src/component/project/Project/PaginatedProjectFeatureToggles/hooks/useRowActions.tsx:1]: This is only if archived filter is on, so only archived flags are displayed.

**Pull request merged**